web/src/components/search/SearchSection.tsx

```diff
@@ -518,6 +518,7 @@ export const SearchSection = ({
 
         <div className="absolute left-0 w-full top-0">
           <FunctionalHeader
+            reset={() => setQuery("")}
```

Comment:
If we're doing this, it should probably be consistent across the different ways to create a new chat? E.g. the `New Chat` button with sidebar expanded should also wipe the query